src/components/Keyword/KeywordModal/index.js

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect } from 'react';
+import styled from 'styled-components';
+import { deleteKeyword } from 'store/keyword';
+import { useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router';
+
+const ModalContainer = styled.div`
+  width: 168px;
+  height: 114px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  position: absolute;
+  left: 205px;
+  top: ${(props) => 75 + props.modalPosition * 43}px;
+  background-color: #fff;
+  border: 1px solid #eee;
+  box-shadow: 0 0 5px 0 rgba(0, 0, 0, 0.15);
+  z-index: 100;
+`;
+
+const KeywordDeleteContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  padding: 0px 12px;
+  margin-bottom: 24px;
+  cursor: pointer;
+`;
+
+const DeleteTitle = styled.span``;
+
+const EditTitie = styled.span``;
```

Comment:
오타가 있네요
```suggestion
const EditTitle = styled.span``;
```

혹시 DeleteTitle 과 EditTitle 스타일이 둘다 동일하다면 모달이니까 그냥 Title로 통일하는게 나을거 같습니다.